test/http_to_do_test.exs

```diff
@@ -39,4 +44,16 @@ defmodule HTTPServerTest.ToDo do
 
     assert API.create(handle_resp) == error_message
   end
+
+  test "Can delete \"{\"todo1\":\"Act\"}\" from data file" do
+    todo_to_be_deleted = "1"
+
+    API.delete(todo_to_be_deleted)
+
+    file_contents =
+      File.read!(@file_path)
+      |> JSON.decode!()
+
+    refute file_contents[todo_to_be_deleted]
+  end
```

Comment:
should there be more tests here? ex: one to verify you can delete the first todo in a list, one to verify you can delete the last todo in a list, one to verify you can delete a todo in the middle of list, one to verify out of bounds ids (< 0, but also > list.size)

Reply:
also, what happens if you try to delete a todo from an 'empty list of todos'

Reply:
I just pushed up a commit that I think covers most if not all of the test cases you highlighted were missing!